R/bcdc-web-services.R

```diff
@@ -32,28 +32,43 @@ bcdc_get_geodata <- function(record = NULL, crs = 3005) {
 #' Query data from the B.C. Web Feature Service
 #'
 #' Queries features from the B.C. Web Feature Service. The data must be available as
-#' a WMS/WFS service. See `bcdc_get_record(record)$resources`). Pulls features off the web.
-#' See `bcdc_get_record(record)$resources`). If the record is greater than 10000 rows,
+#' a WMS/WFS service. See `bcdc_get_record(record)$resources`). If the record is greater than 10000 rows,
```

Comment:
Should we change this from WMS/WFS to just web?

Reply:
Maybe we can merge this and then do a full sweep and change all `WMS/WFS` references to `Web Service`?